### PR TITLE
Unify data reading

### PR DESCRIPTION
**Pull request by KirillTim**
https://github.com/marco-c/crashsimilarity/issues/35
Are there any suggestions for better function name?

### CODE REVIEW

File: crash_similarity.py [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
+import multiprocessing
 import os
-import time
-import download_data
-import utils
 import random
+import time
+
 import gensim
 import smart_open
```
Comment (marco-c) [inline crash_similarity.py]: We can move the usage of `smart_open` in utils.py, this way we can avoid the `open_file_function` argument for `read_files_generator`.
I don't expect to use `smart_open` other than in utils.py, so it should be fine to move it there.

**Comment by marco-c** [comment]
> Are there any suggestions for better function name?

I would name it `read_files` instead of `read_files_generator`, to hide the implementation details.

File: crash_similarity.py (hunk 1 above)
Comment (KirillTim) [inline crash_similarity.py]: Ok. My idea was to make general function for future :)

**Comment by marco-c** [comment]
Minor flake8 issue: https://travis-ci.org/marco-c/crashsimilarity/jobs/212201270#L216.

**Comment by KirillTim** [comment]
done

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**